### PR TITLE
Test setcontent for DBs without json_extract

### PR DESCRIPTION
**Pull request by I-Valchev**

### CODE REVIEW

**Review by bobdenotter — APPROVED** [review]
Neat!

File: public/theme/skeleton/custom/setcontent_1.twig [hunk 1]
```diff
@@ -155,7 +155,7 @@
         {% endfor %}
 
         <ul>
-            <li>Two different random results: [{{ first_ids === second_ids ? "no" : "yes" }}]</li>
+            <li>Two different random results: [{{ first_ids === second_ids ? "[no]" : "[yes]" }}]</li>
```
Comment (bobdenotter) [inline public/theme/skeleton/custom/setcontent_1.twig:158]: Are the double brackets around `[[yes]]` or `[[no]]` intentional?

Comment (I-Valchev) [reply public/theme/skeleton/custom/setcontent_1.twig:158]: nope! definitely not.

**Comment by I-Valchev** [comment]
Close. Will open a new PR with correct changes.

**Pull request closed without merging**